ocx26/index.html

```
@@ -0,0 +1,642 @@
+    </aside>
+  </section>
+
+  <section class="light" data-background-color="#FFFFFF">
+    <h3>The MCP Wrapper</h3>
+    <pre><code class="language-python">from mcp.server.fastmcp import FastMCP
+mcp = FastMCP("tmll-cli-mcp-server")
+
+@mcp.tool()
+def detect_anomalies(experiment_id: str,
+                     keywords: list[str] = None,
+                     method: str = None) -> str:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)